benchmarks/benchmarks/stats.py

```diff
@@ -58,28 +58,9 @@ def time_ttest_ind_diff_var(self):
 
 class Distribution(Benchmark):
     # add distributions here
-    dists = ['alpha', 'anglit', 'arcsine', 'argus', 'beta', 'betaprime',
-             'bradford', 'burr', 'burr12', 'cauchy', 'chi', 'chi2', 'cosine',
-             'crystalball', 'dgamma', 'dweibull', 'erlang', 'expon',
-             'exponnorm', 'exponweib', 'exponpow', 'f', 'fatiguelife',
-             'fisk', 'foldcauchy', 'foldnorm', 'frechet_r', 'frechet_l',
-             'genlogistic', 'gennorm', 'genpareto', 'genexpon', 'genextreme',
-             'gausshyper', 'gamma', 'gengamma', 'genhalflogistic',
-             'geninvgauss', 'gilbrat', 'gompertz', 'gumbel_r', 'gumbel_l',
-             'halfcauchy', 'halflogistic', 'halfnorm', 'halfgennorm',
-             'hypsecant', 'invgamma', 'invgauss', 'invweibull', 'johnsonsb',
-             'johnsonsu', 'kappa4', 'kappa3', 'ksone', 'kstwo', 'kstwobign',
-             'laplace', 'levy', 'levy_l', 'levy_stable', 'logistic',
-             'loggamma', 'loglaplace', 'lognorm', 'loguniform', 'lomax',
-             'maxwell', 'mielke', 'moyal', 'nakagami', 'ncx2', 'ncf', 'nct',
-             'norm', 'norminvgauss', 'pareto', 'pearson3', 'powerlaw',
-             'powerlognorm', 'powernorm', 'rdist', 'rayleigh', 'rice',
-             'recipinvgauss', 'semicircular', 'skewnorm', 't', 'trapz',
-             'triang', 'truncexpon', 'truncnorm', 'tukeylambda', 'uniform',
-             'vonmises', 'vonmises_line', 'wald', 'weibull_min',
-             'weibull_max', 'wrapcauchy', 'bernoulli', 'betabinom', 'binom',
-             'boltzmann', 'dlaplace', 'geom', 'hypergeom', 'logser', 'nbinom',
-             'planck', 'poisson', 'randint', 'skellam', 'zipf', 'yulesimon']
+    dists = ([d[0] for d in distcont + distdiscrete] +
+             ['frechet_l', 'frechet_r', 'trapz', 'laplace_asymmetric',
+              'nhypergeom', 'reciprocal', 'trapezoid'])
```

Comment:
Do you know why these distributions are not in those lists? I suspect that they should be.

Reply:
`['frechet_l', 'frechet_r', `trapz`]` are described as duplicates in of `weibull_max`, `weibull_min` and `trapezoid`, so I removed them. I checked again and realized the two others are in fact in there.

However, `['laplace_asymmetric']` is omitted. It doesn't note anything special about it in the [documentation](https://github.com/scipy/scipy/issues/12924).